### PR TITLE
Feat/l2 bridge scroll (separated to simplify review)

### PR DESCRIPTION
**Pull request by arwer13**

### CODE REVIEW

File: l2-bridge-scroll/src/utils/constants.ts [hunk 1]
```diff
@@ -4,37 +4,40 @@ type ProxyContract = {
...
   L2_ERC20_TOKEN_GATEWAY: L2_ERC20_TOKEN_GATEWAY_TYPE
-  MANTLE_WST_ETH_BRIDGED: MANTLE_WST_ETH_BRIDGED_TYPE
+  SCROLL_WST_ETH_BRIDGED: SCROLL_WST_ETH_BRIDGED_TYPE
   RolesMap: RoleHashToName
 }
 
-const GOV_BRIDGE_ADDRESS: string = '0x3a7b055bf88cdc59d20d0245809c6e6b3c5819dd'
-const L2_ERC20_TOKEN_GATEWAY_ADDRESS: string = '0x9c46560D6209743968cC24150893631A39AfDe4d'
-const MANTLE_WST_ETH_BRIDGED_ADDRESS: string = '0x458ed78EB972a369799fb278c0243b25e5242A83'
+// NB: obfuscate url if the rpc isn't public Buffer.from('???', 'base64').toString('utf-8')
```
Comment (sergeyWh1te) [inline l2-bridge-scroll/src/utils/constants.ts:19]: let's remove this comment

also, we use constants only in one place. They could'be unshared
Or add them into one exportable value - Address

File: l2-bridge-scroll/tests/e2e/scroll_provider.spec.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import { App } from '../../src/app'
 
-describe('MantleProvider', () => {
+describe('ScrollProvider', () => {
   test('fetchBlocks', async () => {
     const app = await App.getInstance()
 
     const start = 50_183_000
     const end = 50_184_000
-    const blocks = await app.mantleClient.fetchBlocks(start, end)
```
Comment (sergeyWh1te) [inline l2-bridge-scroll/tests/e2e/scroll_provider.spec.ts:9]: This test could be pass becasue latest scroll block at current moment - 4_497_161

**Review by sergeyWh1te — CHANGES_REQUESTED** [review]
Need to insert line to github/.codeowners with scroll

File: l2-bridge-scroll/tests/e2e/scroll_provider.spec.ts (hunk 2 above)
Comment (arwer13) [reply l2-bridge-scroll/tests/e2e/scroll_provider.spec.ts:9]: thanks, fixed

**Comment by arwer13** [comment]
Merging to the main feature branch, because this PR was auxiliary.

**Pull request merged**